precompiles/author-mapping/src/lib.rs

```diff
@@ -68,7 +67,7 @@ where
 			Action::AddAssociation => Self::add_association(handle),
 			Action::UpdateAssociation => Self::update_association(handle),
 			Action::ClearAssociation => Self::clear_association(handle),
-			Action::RegisterKeys => Self::register_keys(handle),
+			Action::RemoveKeys => Self::remove_keys(handle),
```

Comment:
:warning: Here's another breaking change (I didn't see it called out explicitly in the PR desc)

Reply:
The first breaking change says we remove `RegisterKeys` to use `SetKeys` instead

The last breaking change says we deprecate `ClearAssociation` and add `RemoveKeys`

Reply:
Yeah, I'm pointing out that this affects precompiles